src/protocol/tx_validity.md

```diff
@@ -183,3 +183,14 @@ If change outputs are present, they must have:
 ### State Changes
 
 Transaction processing is completed by removing spent UTXOs from the state and adding created UTXOs to the state.
+
+### Coinbase
+
+The coinbase is a mechanism for block creators to convert fees into spendable UTXOs.
+
+In order for a coinbase to be valid:
+
+1. It must issued in the form of a [Mint](../protocol/tx_format/transaction.md#TransactionMint) transaction.
+2. The coinbase must be the first transaction within a block.
+3. The total output value of the coinbase cannot exceed the total amount of fees processed from all other transactions within the same block.
+4. The `asset_id` for coinbase outputs must match the `asset_id` that fees are paid in (`asset_id == 0`).
```

Comment:
```suggestion
4. The `asset_id` for coinbase outputs must match the `asset_id` that fees are paid in (`asset_id == 0` by default).
```

Reply:
I wouldn't say this is a default; rather, it's a requirement that the asset ID of the coinbase tx output MUST be something. That something could change later, but it's still a requirement.

Reply:
I meant that the default asset to pay fee is `0`=) But it can be different for different networks in the future(I don't know why, but who knows)

Reply:
closing this thread as we use asset_id = 0 widely throughout the specs to refer to the base asset. If we want to make this more configurable that would be a larger, separate change.